x11-wm/mutter/metadata.xml

```diff
@@ -9,6 +9,9 @@
     <flag name="elogind">Rely on <pkg>sys-auth/elogind</pkg> as logind provider for Wayland sessions</flag>
     <flag name="libdisplay">Enable EDID and DisplayID using <pkg>media-libs/libdisplay-info</pkg></flag>
     <flag name="sysprof">Enable profiling data capture support using <pkg>dev-util/sysprof-capture</pkg></flag>
+    <flag name="wayland">Enable wayland support</flag>
+    <flag name="X">Enable X11 support</flag>
```

Comment:
Adding wayland and X USE descriptions here is not really needed as their descriptions are nearly the same as those of the global flags -> use.desc